tests/Fixtures/JunitFormat/features/bootstrap/SkippedTestCasesContext.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Behat\Behat\Context\Context;
+use Behat\Hook\BeforeScenario;
+use Behat\Step\Given;
+use Behat\Step\Then;
+
+class SkippedTestCasesContext implements Context
+{
+    private $value;
+
+    #[BeforeScenario]
```

Comment:
This might be a step too far, but again it *could* be possible to implement this as a tagged hook, so that it is always present in the context but only triggered for the feature that actually needs it?